src/utils/bitmap.rs

```diff
@@ -67,13 +67,38 @@ fn load_from_png_file(filename: &str) -> Bitmap {
     };
 }
 
+/// # Panics
+///
+/// Will panic if atleast one pixel has incorrect rgb values
+#[must_use]
+pub fn to_grayscale(bitmap: &Bitmap) -> Bitmap {
```

Comment:
I thought you want to mutate, if copying it's better to create specific bitmap as Gray-scale bitmap, without RGBA, just L